lib/commands/general.js

```diff
@@ -228,7 +228,7 @@ commands.reset = async function () {
   await androidHelpers.resetApp(this.adb, Object.assign({}, this.opts, {fastReset: true}));
   // reset context since we don't know what kind on context we will end up after app launch.
   this.curContext = NATIVE_WIN;
-
+  await this.stopChromedriverProxies();
```

Comment:
What about calling `switchContext(NATIVE_WIN)`?
As you mentioned in your side note, if we set native window, the method calls _stop_ or _suspend_ chrome driver properly..?

Reply:
Sure. I was just following the pattern from `closeApp` in the same file. If we call `switchContect` we get the added benefit of respecting the `recreateChromeDriverSessions` desired cap.
On `closeApp` are we ok leaving chromedriver still running if somebody set this cap?